posix.mak

```diff
@@ -130,14 +108,16 @@ $(CURL_TOOLS): $(ROOT)/%: %.d
 $(TOOLS) $(DOC_TOOLS): $(ROOT)/%: %.d
 	$(DMD) $(MODEL_FLAG) $(DFLAGS) -of$(@) $(<)
 
-$(TAGS): %.tag: $(DOC)/%.html $(ROOT)/findtags
-	$(ROOT)/findtags $< > $@
+$(DOC)/chmgen : $(DOC)/chmgen.d
+	$(DMD) -g -of$@ $<
+
+ALL_OF_PHOBOS_DRUNTIME_AND_DLANG_ORG = # ???
```

Comment:
1. There is no SPEC_ROOT in tools/posix.mak or tools/win32.mak.
2. How am I supposed to get the "druntime and phobos documentables" without three pages of copy-pasta?